### PR TITLE
reports: add 2024 Q1 report

### PR DESCRIPTION
**Pull request by frasertweedale**
---

## Advisory

- [ ] It's not duplicated
- [ ] All fields are filled
- [ ] It is validated by `hsec-tools`

## hsec-tools

- [ ] Previous advisories are still valid

### CODE REVIEW

File: reports/2024-04-xx-Q1-report.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+that are widely depended upon.  Issue
+[#162](https://github.com/haskell/security-advisories/issues/162)
+tracks this topic.  We welcome all community members to contribute
+to the discussion.  The SRT will communicate further about this need
+in the coming months.
+
+
+## SRT at Haskell Ecosystem Workshop and ZuriHac 2024
+
+In early June, members (TODO who?) of the SRT will attend the
```
Comment (blackheaven) [inline reports/2024-04-xx-Q1-report.md:87]: I should be attending too (I'm waiting for my ticket)

**Review by blackheaven — APPROVED** [review]

**Review by TristanCacqueray — APPROVED** [review]

**Comment by frasertweedale** [comment]
Thanks for reviews.  I will wait for the CI recommendations section to be contributed by @mihaimaruseac.

**Review by blackheaven — APPROVED** [review]

**Comment by frasertweedale** [comment]
@mihaimaruseac I want to publish the report on Monday.  If you are able to prepare a CI recommendations section by then, great.  If not, no big deal, we can include in the next report or make a standalone post on Discourse about it.

**Comment by frasertweedale** [comment]
I published the report: https://discourse.haskell.org/t/haskell-security-response-team-2024-january-march-report/9285.

Merging this PR.

**Comment by mihaimaruseac** [comment]
Sorry, completely missed this. Will make a new PR and we can post on Discourse after that.

**Pull request merged**